src/jit/lsraxarch.cpp

```diff
@@ -3698,6 +3216,84 @@ bool Lowering::ExcludeNonByteableRegisters(GenTree* tree)
 }
 #endif // _TARGET_X86_
 
+//------------------------------------------------------------------------
+// GetIndirSourceCount: Get the source registers for an indirection that might be contained.
+//
+// Arguments:
+//    node      - The node of interest
+//
+// Return Value:
+//    The number of source registers used by the *parent* of this node.
+//
+int Lowering::GetIndirSourceCount(GenTreeIndir* indirTree)
+{
+    int      srcCount = 0;
```

Comment:
Actually, they are identical, so I'll just move it into lower.cpp. (Eventually it probably belongs somewhere else since all of the `Lowering::TreeNodeInfoInit` stuff will be done during LinearScan - but for now at least I'll make them common.)